cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -132,7 +138,8 @@ public enum LogicalType {
     TIME_MILLIS(Type.INT, "time-millis"),
     TIME_MICROS(Type.LONG, "time-micros"),
     DECIMAL(Type.BYTES, "decimal"),
-    DATETIME(Type.STRING, "datetime");
+    DATETIME(Type.STRING, "datetime"),
+    FIXED(Type.BYTES, "fixed");
```

Comment:
Adding any new type would require testing across all the plugins. It does not matter if it's physical or logical. How the schema would look from UI? How would all the sinks handle it? How about plugins? Any change to schema API should go through design review

Reply:
I told Uma to create a UI ticket after this platform change is merged.

Most plugins should be able to work just fine using the underlying BYTES base type, so I don't expect anything to break there.

If a plugin wants to explicitly handle FIXED types, the plugin can implement logic to handle this.

Reply:
OK. Then we should have a set of e2e tests for all our tier-1 plugins. We had a lot of problems with different plugins not supporting decimals properly, we need to avoid it here

Reply:
I will create a separate bug for e2e tests for plugins